include/hal/library/responder/measlib.h

```diff
@@ -149,6 +149,23 @@ extern bool libspdm_generate_measurement_summary_hash(
     uint8_t measurement_summary_hash_type,
     uint8_t *measurement_summary_hash,
     uint32_t measurement_summary_hash_size);
+
+/**
+ * Collect the measurement extension log.
+ *
+ * @param  spdm_mel               Return the pointer of MEL.
+ * @param  spdm_mel_szie          Return the size of MEL.
+ * @param  measurement_hash_algo  Indicates the measurement hash algorithm.
+ * Must be SPDM_ALGORITHMS_MEASUREMENT_HASH_ALGO_* value in spdm.h.
+ *
+ * @retval true  measurement summary hash is generated or skipped.
+ * @retval false measurement summary hash is not generated.
+ **/
+bool libspdm_measurement_extension_log_collection(
```

Comment:
Pass the `spdm_context` as a parameter.

Reply:
Thanks. I have added the `spdm_context` parameter.

Reply:
The `LIBSPDM_HAL_PASS_SPDM_CONTEXT` is not needed since this a new function. The macro is only for already published functions.

Reply:
Sorry. I didn't know the history of LIBSPDM_HAL_PASS_SPDM_CONTEXT before. Now I have deleted the MACRO.